### DISCUSSION TITLE
Is there a possibility to turn off full screen switch in the Viewer

### DISCUSSION THREAD

**Discussion by r-pankevicius** [RussCoder/djvujs#45] [Q&A] [answered]
Hi, @RussCoder  !

Is there such possibility to hide that full screen switch by passing some option to the new DjVu.Viewer() constructor?

If page is designed this way that viewer takes 100%, then this switch is useless.

/remis

**Comment by r-pankevicius** [comment]
Or maybe more simple trick: add CSS class for component full-page-switch that is not mentioned in djvu_viewer.css, thus has no effect.
But someone who embeds the viewer can use that full-page-switch css class to set visibility to none.

**Comment by RussCoder** [comment, marked as answer]
Hello!

No, there is no such a possibility right now. If you need it I can add it as an option to the `configure` method of the viewer. I think it's better than a CSS class - you are not the first who expressed a desire to adjust the UI a bit.

**Reply by RussCoder** [reply]
Now you can pass 
```
    uiOptions: {
        hideFullPageSwitch: true,  
    },
```
to the `configure()` method.
